fantasy_football_project/urls.py

```diff
@@ -42,23 +42,47 @@
     PlayerListAPIView,
 )
 
+from fantasy_football_app.auth_views import (
+    SignupView,
+    LoginView,
+    LogoutView,
+    AuthStatusView,
+    ForgotPasswordView,
+    ConfirmForgotPasswordView,
+    ChangePasswordView,
+    CSRFTokenView
+)
+
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', index, name='index'),  # This line maps the root path to the index view
-    path('register/', register, name='register'),
-    path('sign_in/', sign_in, name='sign_in'),
-    path('user_home/', user_home, name='user_home'),
-    path('delete_entry/<int:entry_id>/', delete_entry, name='delete_entry'),
-    path('standings/', standings, name='standings'),
-    path('view_entry/<int:entry_id>/', view_entry, name='view_entry'),
-    path('sign_out/', sign_out, name='sign_out'),
-    path('reset_password/', auth_views.PasswordChangeView.as_view(template_name='fantasy_football_app/registration/password_change.html'), name='password_change'),
-    path('password_change/done/', auth_views.PasswordChangeDoneView.as_view(template_name='fantasy_football_app/registration/password_change_done.html'), name='password_change_done'),
-    path('players/', players_view, name='players'),
-    path('rules/', rules, name='rules'),
-    path('player/<int:player_id>/', player_stats_view, name='player_stats'),
-    path('entry_list/', entry_list_view, name='entry_list'),
+
+    # path('', index, name='index'),  # This line maps the root path to the index view
```

Comment:
You can just remove these. If we need them later we can just look back at the branch history